src/components-examples/material/table/table-reorderable/table-reorderable-example.ts

```diff
@@ -0,0 +1,39 @@
+import {Component} from '@angular/core';
+import {CdkDragDrop, moveItemInArray} from '@angular/cdk/drag-drop';
+
+/**
+ * @title Table with re-orderable columns
+ */
+@Component({
+  selector: 'table-reorderable-example',
+  templateUrl: './table-reorderable-example.html',
+  styleUrls: [ './table-reorderable-example.css' ]
```

Comment:
There shouldn't be spaces inside the array literal.

Reply:
Done